### PR TITLE
[397] disable token modal and filter tokens

### LINKED ISSUE #397: [Product] Prevent opening asset list unless wallet is connected
If no wallet is connected in the FROM section, the asset list should not open when the user taps the Asset component.

When the list is open, it should show the tokens in the wallet.

To discuss: how should tokens that are not present in the config be displayed, if in any way? should we only display the tokens in the list?

For now: just prevent opening the asset list as decribed above and expand the list of supported tokens with USDC and others defined by @aakashthumaty

### PR DESCRIPTION
**Pull request by anondev2323**
Closes #397

### CODE REVIEW

File: wormhole-connect/src/views/Bridge/Modals/TokensModal.tsx [hunk 1]
```diff
@@ -363,13 +362,6 @@ function TokensModal() {
           </Collapse> */}
         </div>
       </Scroll>
-
-      <div className={classes.register}>
-        <div className={classes.registerText}>Don't see your token?</div>
-        <a href="#" className={classes.registerLink}>
-          Register token
-        </a>
-      </div>
```
Comment (anondev2323) [inline wormhole-connect/src/views/Bridge/Modals/TokensModal.tsx:372]: Really shouldn't prompt user to register tokens here, kind of a dumb ux. Can just show the "no results" for tokens with balances

File: wormhole-connect/src/views/Bridge/Modals/TokensModal.tsx [hunk 2]
```diff
@@ -255,19 +255,18 @@ function TokensModal() {
...
-              tokenBalances[t.symbol] !== null,
-          )
-        : networkTokens;
+    if (!fromNetwork || !walletAddr) {
+      setTokens(networkTokens);
+      return;
+    }
+    const filtered = networkTokens.filter((t) => {
+      const b = tokenBalances[t.symbol];
+      return b !== null && b !== '0';
```
Comment (anondev2323) [inline wormhole-connect/src/views/Bridge/Modals/TokensModal.tsx:266]: Should not filter out balances of `undefined`.  This means it just hasn't gotten the balance yet and will display a spinner next to that token while it is fetching.

**Review by M-Picco — APPROVED** [review]

**Comment by anondev2323** [comment]
Closed #376

**Comment by anondev2323** [comment]
I think this resolves #352 also

**Comment by anondev2323** [comment]
Closes #258

**Comment by anondev2323** [comment]
Closes #242

**Comment by anondev2323** [comment]
Seems like clearing the balances on network change has a lot of downstream affects/improvements

**Pull request merged**